### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+language: ruby
+sudo: false
+cache: bundler
+script:
+- bundle exec rake
+rvm:
+- 2.5.3
```
Comment (ekohl) [inline .travis.yml]: Any reason to choose 2.5.3 here instead of `'2.5'` so it's the latest 2.5?

File: voxpupuli-release.gemspec [hunk 2]
```diff
@@ -19,4 +19,5 @@ Gem::Specification.new do |s|
   # Runtime dependencies, but also probably dependencies of requiring projects
   s.add_runtime_dependency 'rake'
   s.add_runtime_dependency 'puppet-blacksmith', '>= 4.0.0'
+  s.add_development_dependency 'github_changelog_generator'
```
Comment (ekohl) [inline voxpupuli-release.gemspec]: This is the very old released version from 2016. Is that new enough?

File: Rakefile [hunk 3]
```diff
@@ -0,0 +1,17 @@
+$LOAD_PATH.unshift File.expand_path('../lib', __FILE__)
```
Comment (ekohl) [inline Rakefile]:
```suggestion
$LOAD_PATH.unshift File.expand_path('lib', __dir__)
```

File: .travis.yml (hunk 1 above)
Comment (bastelfreak) [reply .travis.yml]: 2.5.3 is bundled in the travis image. so the run is a bit faster. if we set it to 2.5, travis will download the ruby version on each run.

File: voxpupuli-release.gemspec (hunk 2 above)
Comment (bastelfreak) [reply voxpupuli-release.gemspec]: I tested it and it worked fine in this repository.

**Comment by bastelfreak** [comment]
@ekohl can you review again and approve?

**Pull request merged**